src/Gaufrette/Adapter/Local/composer.json

```diff
@@ -0,0 +1,31 @@
+{
+    "name": "gaufrette/local-adapter",
```

Comment:
if you also want to expose it as its own package, you must have the replace rule in the main composer.json too, otherwise things will behave weird.

And providing subtree splits for adapters does not make any sense if the core does not have its own subtree split as well. The adapter package must not depend on the full repo (as the full repo also contains the adapter code)